pixl_imaging/tests/test_imaging_processing.py

```diff
@@ -171,6 +229,107 @@ async def test_querying_without_uid(orthanc_raw, caplog) -> None:
     assert await study.query_local(orthanc)
 
     expected_msg = (
-        f"No study found with UID {study.message.study_uid}, trying MRN and accession number"
+        f"No study found in modality UCVNAQR with UID {study.message.study_uid}, "
+        "trying MRN and accession number"
+    )
+    assert expected_msg in caplog.text
+
+
+class MockDateTime(datetime.datetime):
+    """Mock the date and time to be a Monday at 2 am."""
+
+    @classmethod
+    def now(cls, tz=None) -> datetime.datetime:
+        return MockDateTime(2024, 1, 1, 2, 0, tzinfo=tz)
+
+
+@pytest.mark.processing()
+@pytest.mark.asyncio()
+@pytest.mark.usefixtures("_add_image_to_fake_vna")
```

Comment:
can probably remove using this fixture so that the VNA has no data in it at all. I'm assuming that the teardown here removes the data from the VNA?

Reply:
ah yeah that would definitely make testing easier

Reply:
I've removed the unused fixtures